### PR TITLE
Ynikz branch

### PR DESCRIPTION
**Pull request by ynikki**
Added a template page for the views, created routes folder, and added a template.js

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,9 +2,13 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Hi</title>
+  <title>inHue</title>
+  <link rel="stylesheet" type="text/css" href="/scss/styles.scss">
```
Comment (thgaskell) [inline public/index.html]: You can't include scss as a stylesheet.

File: public/views/template.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>inHue</title>
+  <link rel="stylesheet" type="text/css" href="/scss/styles.scss">
```
Comment (thgaskell) [inline public/views/template.html]: You can't include scss as a stylesheet.

File: public/index.html (hunk 1 above)
Comment (ynikki) [inline public/index.html]: Thanks @thgaskell I changed it.

File: public/views/template.html (hunk 2 above)
Comment (ynikki) [inline public/views/template.html]: Thanks @thgaskell I changed it.

**Comment by ynikki** [comment]
Fixes #9

**Comment by thgaskell** [comment]
LGTM 👍

**Pull request merged**